download-data.py

```diff
@@ -54,35 +371,18 @@ def main() -> None:
         " To switch back, rerun this script with the `--release current` option.",
     )
     parser.add_argument(
-        "--format",
+        "--module-results",
```

Comment:
Is this argument intended to work with `--format`? Because it doesn't appear to be respected in my testing, and the output of `--help` lead me to believe it would be supported.

Reply:
No, it isn't, because I what format the results might take seemed too hard to predict. In this case we have both SCE and Anndata, but I don't expect it to be the norm, so I wasn't sure what to do to handle it at this stage. Maybe a future enhancement to allow more selectivity from results downloads?

Reply:
Then the help text should say something about:

> These results can be refined by project or sample, but not by other filters, as I didn't want to make assumptions about file names.

Rather than:

>  When this option is selected, the `--process-stage` and `--include-reports` options are ignored